scripts/deploy-cursorless-nvim.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+# This script is used to push to the cursorless.nvim github production repo
+set -euo pipefail
+
+# Clone current cursorless.nvim main
+mkdir -p dist && cd dist
+git clone 'https://github.com/hands-free-vim/cursorless.nvim.git' cursorless.nvim-remote
+cd -
+
+out_dir=dist/cursorless.nvim-remote
+
+# Delete the old files
+cd "$out_dir"
+git rm -r README.md lua/ vim/ node/
+cd -
+
+#
+# Merge the build .js and the static files
+#
+
+# copy .lua, .vim dependencies, command-server and other static files
+cp -r cursorless.nvim/README.md "$out_dir/"
+cp -r cursorless.nvim/lua "$out_dir/"
+cp -r cursorless.nvim/vim "$out_dir/"
+cp -r cursorless.nvim/node "$out_dir/"
```

Comment:
why not just copy everything? I've added a few things which it would be nice not to have to list here

Reply:
You are right, we can copy everything, especially because we don't create the symlinks on CI. Do you mind fixing it while testing the production deployment too?

Reply:
The only way to test the production deployment is to merge the PR. If it breaks, we'll fix in a follow-up